### ISSUE TITLE
How would the class IsAuthenticated work after recieving jwt from server?

### ISSUE THREAD

**Issue by shubh3794** [GetBlimp/django-rest-framework-jwt#217]
I have recieved JWT from the server and after that I am attaching a header
Authorization : JWT <token>
to every outgoing request
On the backend I want to restrict some views and features to only authenticated users
Can  drf's permission class IsAuthenticated achieve this for me?

**Comment by vaibhav-jain** [comment]
@shubh3794 yes absolutely but you have to add `JWT` Authentication class to `REST_FRAMEWORK    DEFAULT_AUTHENTICATION_CLASSES settings` after that whenever you want to restrict any views simply specify permission class `IsAuthenticated` . Or else you can specify `Authentication` class in view too. DRF's documents has examples. In short yes you have to explicitly specify the permission class at view level otherwise anyone can access the view.

**Comment by towry** [comment]
It's not working
